src/modules/Expanders.jl

```diff
@@ -302,6 +400,18 @@ function Selectors.runner(::Type{DocsBlocks}, x, page, doc)
 
         # Update `doc` with new object and anchor.
         docstr   = get(filtered)
+
+        # If there are any methods then we add a segment to the docstring.
+        # The markdown content of the docstring is actually the first element
+        # in the docstring content, and we push to that vector.
+        # Also, since there are differences between 0.4 and 0.5, we only support
+        # 0.5 at this point. There is not table generated for 0.4.
+        if VERSION > v"0.5-"
```

Comment:
I prefer to keep `VERSION` checks at toplevel so they're easier to find when removing them later. So rather add a mock `methodtable_md` definition when on version `0.4`:

```julia
if VERSION < v"0.5.0-dev"
    function methodtable_md(...)
        # the real def
    end
else
    methodtable_md(...) = Nullable{Markdown.Paragraph}()
end
```